src/gradata/hooks/session_persist.py

```
@@ -0,0 +1,103 @@
+        pass
+    return None
+
+
+def _get_modified_files() -> list[str]:
+    """Get files modified + untracked in current session via git."""
+    cwd = os.environ.get("CLAUDE_PROJECT_DIR", ".")
+    files = []
+
+    # Staged and unstaged changes
+    try:
+        result = subprocess.run(
```

Ruff (0.15.9)

[warning] `subprocess.run` without explicit `check` argument

Add explicit `check=False`

(PLW1510)